Documentation/kubernetes-on-aws.md

```diff
@@ -68,210 +81,21 @@ aws_access_key_id = AKID1234567890
-
-* **credentials/**
-
-  This directory contains the **unencrypted** TLS assets for your cluster, along with a pre-configured `kubeconfig` file which provides access to your cluster api via kubectl.
-
-You can also now check the `my-cluster` asset directory into version control if you desire. The contents of this directory are your reproducible cluster assets. Please take care not to commit the `my-cluster/credentials` directory, as it contains your TLS secrets. If you're using git, the `credentials` directory will already be ignored for you.
-
-### Optional Calico network policy
-
-The cluster can be configured to use Calico to provide network policy.
-
-Edit the `cluster.yaml` file:
-```yaml
-useCalico: true
-kubernetesVersion: v1.2.4_coreos.cni.1
-```
-The hyperkube image version needs to contain the CNI binaries (these are tagged with `_cni`)
-
-### Optional Route53 Host Record
-
-`kube-aws` can optionally create an A record for the controller IP in an existing hosted zone.
-
-Edit the `cluster.yaml` file:
-
-```yaml
-externalDNSName: my-cluster.staging.core-os.net
-createRecordSet: true
-hostedZone: staging.core-os.net
-```
-
-If `createRecordSet` is not set to true, the deployer will be responsible for making externalDNSName routable to the controller IP after the cluster is created.
-
-### Validate cluster assets
-
-The `validate` command check the validity of the cloud-config userdata files and the CloudFormation stack description:
-
-```sh
-$ kube-aws validate
-```
-
-### Create a cluster from asset directory
-
-Now for the exciting part, create your cluster:
-
-```sh
-$ kube-aws up
-```
-
-**PRODUCTION NOTE**: the TLS keys and certificates generated by `kube-aws` should *not* be used to deploy a production Kubernetes cluster.
-Each component certificate is only valid for 90 days, while the CA is valid for 365 days.
-If deploying a production Kubernetes cluster, consider establishing PKI independently of this tool first.
-
-### Configure DNS
-
-If you configured Route 53 settings in your configuration above via `createRecordSet`, a host record has already been created for you.
-
-Otherwise, navigate to the DNS registrar hosting the zone for the provided external DNS name. Ensure a single A record exists, routing the value of `externalDNSName` defined in `cluster.yaml` to the externally-accessible IP of the master node instance.
-
-You may use `kube-aws status` to get this value after cluster creation, if necessary. This command can take a while.
-
-### Access the cluster
+## Test Credentials
 
-A kubectl config file will be written to a `kubeconfig` file, which can be used to interact with your Kubernetes cluster like so:
+Test that your credentials work by describing any instances you may already have running on your account:
```

Comment:
I bet that most users will have instances on their account, at least in this phase of the adoption curve.